users/tests/test_user_model.py

```diff
@@ -0,0 +1,110 @@
+import secrets
+import string
+
+import pytest
+from django.contrib.auth import get_user_model
+from django.test import Client
+from django.urls import reverse_lazy
+
+User = get_user_model()
+
+HTTP_SUCCESS = 200
+
+
+def generate_random_password(length: int = 12) -> str:
+    characters = string.ascii_letters + string.digits + string.punctuation
+    secure_random = secrets.SystemRandom()
+    return "".join(secure_random.choice(characters) for _ in range(length))
+
+
+@pytest.fixture()
+def generated_password() -> str:
+    return generate_random_password()
+
+
+@pytest.fixture()
+def user(generated_password: str) -> User:
+    user = User.objects.create_user(email="test@example.com", password=generated_password)
+    user.plain_password = generated_password
+    return user
+
+
+@pytest.fixture()
+def admin_user(generated_password: str) -> User:
+    admin_user = User.objects.create_superuser(email="admin@example.com", password=generated_password)
+    admin_user.plain_password = generated_password
+    return admin_user
+
+
+@pytest.fixture()
+def client() -> Client:
+    return Client()
+
+
+@pytest.mark.django_db()
```

Comment:
```suggestion
@pytest.mark.django_db
```
No need for brackets: https://pytest-django.readthedocs.io/en/latest/database.html#enabling-database-access-in-tests
Please remove everywhere unless parameters passed.